hosts_example.yml

```diff
@@ -189,6 +189,21 @@ all:
     # rbac_component_additional_system_admins:
     #   - user1
     #   - user2
+    ##
+    ####  Configuring Role Based Access Control with a remote MDS ####
+    ## To have CP-Ansible configure Brokers and Components for RBAC with the MDS on a remote cluster, set these mandatory variables in addition to those listed above:
+    # rbac_enabled: true
+    # external_mds_enabled: true
+    # mds_ssl_enabled: true
+    ## The URL and Port for the client listener on your Kafka Cluster hosting the MDS
+    # mds_broker_bootstrap_servers: <mds-broker-hostname><port>,<mds-broker-hostname><port>
+    ## The URL for the MDS REST API on your Kafka Cluster hosting MDS
+    # mds_bootstrap_server_urls: http(s)://<mds-broker-hostname>:8090,http(s)://<mds-broker-hostname>:8090
+    # mds_broker_listener:
+    #   ssl_enabled: true <set to false if remote MDS does not use TLS>
+    #   ssl_mutual_auth_enabled: true <set to false if remote MDS doe not use MTLS>
+    #   sasl_protocol: none <set protocol for remote MDS, options are: kerberos, sasl_plain, sasl_scram>
+    ##
```

Comment:
```suggestion
    ####  Configuring Role Based Access Control with a remote MDS ####
    ## To have CP-Ansible configure Brokers and Components for RBAC with the MDS on a remote kafka cluster, set these mandatory variables in addition to those listed above:
    # rbac_enabled: true
    # external_mds_enabled: true
    ## The URL for the MDS REST API on your Kafka Cluster hosting MDS
    # mds_bootstrap_server_urls: http(s)://<mds-broker-hostname>:8090,http(s)://<mds-broker-hostname>:8090
    # mds_ssl_enabled: <set to true if mds_bootstrap_server_urls is https otherwise false>
    ## The listener on the MDS Kafka Cluster to connect to in the next variables
    # mds_broker_bootstrap_servers: <mds-broker-hostname><port>,<mds-broker-hostname><port>
    # mds_broker_listener:
    #   ssl_enabled: true <set to false if remote MDS does not use TLS>
    #   ssl_mutual_auth_enabled: true <set to false if remote MDS doe not use MTLS>
    #   sasl_protocol: none <set protocol for remote MDS, options are: kerberos, sasl_plain, sasl_scram>
    ##
```